streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -577,6 +575,12 @@ class StreamingContext private[streaming] (
    * @throws IllegalStateException if the StreamingContext is already stopped.
    */
   def start(): Unit = synchronized {
+  /**
+   * Registering Streaming Metrics at the start of the StreamingContext
```

Comment:
I think it is better to use `//` rather than `/** */` for internal code commenting.